base/applications/notepad/notepad.h

```diff
@@ -16,6 +16,7 @@
 #include <commdlg.h>
 #include <tchar.h>
 #include <malloc.h>
+#include <stdlib.h>
```

Comment:
For which new functionality do you need this header?

Reply:
For `_countof` macro. Or should I use `ARRAY_SIZE` instead?

Reply:
If `_countof` available, use it.